spell.ignore

```diff
@@ -741,3 +741,4 @@ crit
 err
 getpcaps
 libcap
+GiB
```

Comment:
For the sake of bisectability, this should be on the commit that introduced it, that is c2fe224c58be812bc799d54e3c6aef7ba77603b3.